one_fm/hooks.py

```diff
@@ -249,6 +249,9 @@
 		"on_update": "one_fm.one_fm.utils.send_notification_to_grd_or_recruiter",
 		"after_insert": "one_fm.hiring.utils.after_insert_job_applicant"
 	},
+	"Onboard Employee": {
+		"validate": "one_fm.utils.validate_onboard_employee",
```

Comment:
We can validate it form the Onboard Employee validate method. No need to add hook for it